cmd/minikube/cmd/start.go

```diff
@@ -432,13 +432,23 @@ func displayEnviron(env []string) {
 	}
 }
 
-func showKubectlInfo(kcs *kubeconfig.Settings, k8sVersion string, machineName string) error {
+func showKubectlInfo(kcs *kubeconfig.Settings, k8sVersion, machineName string) error {
 	if k8sVersion == constants.NoKubernetesVersion {
 		register.Reg.SetStep(register.Done)
 		out.Step(style.Ready, "Done! minikube is ready without Kubernetes!")
-		out.BoxedWithConfig(box.Config{Py: 1, Px: 4, Type: "Round", Color: "Green"}, style.Tip, "Things to try without Kubernetes ...", `- "minikube ssh" to SSH into minikube's node.
+
+		// Runtime message.
+		msg := `- "minikube ssh" to SSH into minikube's node.`
+		switch viper.GetString(containerRuntime) {
+		case constants.DefaultContainerRuntime:
+			msg += `
 - "minikube docker-env" to point your docker-cli to the docker inside minikube.
-- "minikube image" to build images without docker.`)
+- "minikube image" to build images without docker.`
+		case constants.CRIO:
+			msg += `
+- "minikube podman-env" to point your podman-cli to the podman inside minikube`
+		}
```

Comment:
We could get rid of some string duplication if we used templating
```
"minikube {{.runtime}}-env" to ...
{{if eq .runtime "docker"}}"minikube image" to ...{{end}}
``` 
```

Reply:
for sake translations we can skip this